src/modm/driver/pressure/ms5611_data.hpp

```diff
@@ -0,0 +1,175 @@
+// coding: utf-8
+// ----------------------------------------------------------------------------
+/*
+ * Copyright (c) 2021, Rasmus Kleist Hørlyck Sørensen
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_MS5611_DATA_HPP
+#define MODM_MS5611_DATA_HPP
+
+namespace modm
+{
+
+// Forward declaration of the driver class
+template <typename SpiMaster, typename Cs>
+class Ms5611;
+
+namespace ms5611data
+{
+
+/**
+ * @brief Holds the factory calibration data from the PROM.
+ * Upon initialization the driver automatically reads the calibration values from the PROM
+ */
+struct modm_packed
```

Comment:
```suggestion
/// @ingroup modm_driver_ms5611
struct modm_packed
```
Otherwise it'll get lost somewhere in the depths of Doxypress…